### ISSUE TITLE
JetBrains: Mac multitouch zoom

### ISSUE THREAD

**Issue by philipp-spiess** [sourcegraph/sourcegraph#33423]
Depends on the Mac prototype: Let's look into differences in pinch to zoom. Should we disable it for the WebView?

**Comment by sourcegraph-bot-2** [comment]
Heads up @muratsu @jjinnii @ryankscott - the "team/growth-and-integrations" label was applied to this issue.

**Comment by vdavid** [comment]
Okay, so this was an easy one. It just works. The web view doesn't zoom in and out by default, which is what we want.
[Half-min Loom vid](https://www.loom.com/share/4fa374e1906742989b5bed7c337f9f45) demonstrating that this is the case.

**Comment by philipp-spiess** [comment]
@vdavid 👀👀👀 what did you do to make it work on a Mac?

**Comment by vdavid** [comment]
@philipp-spiess That was the tricky part. I ran `gradle clean` then `gradle build`, then there was a file at `build/distributions/Sourcegraph-1.2.3.zip`. I had to drag and drop this file to a standard JetBrains IDE instance (I used an IntelliJ Community Edition) to install/update it. After a restart, it worked.
Our friend in the JetBrains team recommended this method for testing on a Mac while the JCEF component doesn't work in sandboxed IDE instances.
It's much less convenient than just running `runIde`, and debugging would be very manual, but it works for running the plugin.
